data_steward/analytics/cdr_ops/rt_cdr_qc/cdr_deid_qa_report6_fitdata.py

```diff
@@ -358,16 +358,112 @@
 
 result = execute(client, query)
 
-if sum(result['bad_rows']) == 0:    
-    summary = summary.append(        {
-            'query': 'Query7 device_id was deidentified properly for all records.',
+if sum(result['bad_rows']) == 0:
+    summary = summary.append(
+        {
+            'query':
+                'Query7 device_id was deidentified properly for all records.',
+            'result':
+                'PASS'
+        },
+        ignore_index=True)
+else:
+    summary = summary.append(
+        {
+            'query':
+                'Query7 device_id was not deidentified properly. See query description for hints.',
+            'result':
+                'Failure'
+        },
+        ignore_index=True)
+result
+
+# -
+
+# # Check de-identification of src_ids.
+#
+# DC-3376
+#
+# This check verifies that src_ids in fitbit tables were updated successfully. The check passes if no records are returned.
+
+# +
+src_id_check = JINJA_ENV.from_string("""
+SELECT  
+distinct person_id,
+SUM(records) as bad_src_id_match,
+'{{table}}' as table,
+FROM (
+    SELECT 
+      distinct person_id,
+      src_id,
+      count(person_id) as records
+    FROM 
+      `{{project}}.{{non_deid_fitbit}}.{{table}}`  
+    group by 1,2
+   
+   union all
+   
+  SELECT 
+    distinct person_id, 
+    src_id 
+    , count(person_id) as records
+  FROM 
+    `{{project}}.{{deid_cdr_fitbit}}.{{table}}` 
+  group by 1,2
+) ft
```

Comment:
I don't think you need to union two tables here. Let me know what you think of this:
```
SELECT  
distinct person_id,
records as bad_src_id_match,
'{{table}}' as table,
FROM (
  SELECT 
    distinct person_id, 
    src_id 
    , count(person_id) as records
  FROM 
    `{{project}}.{{deid_cdr_fitbit}}.{{table}}` 
  group by 1,2
) ft
```

Reply:
It doesn't return anything.